app/controllers/authority/finder_controller.rb

```diff
@@ -0,0 +1,32 @@
+module Authority
+  class FinderController < ApplicationController
+    def search
+      result = map_result(Finder.all_things(params[:q],params[:model]))
+      render json:  result
+    end
+
+    def get_obj
```

Comment:
We should, methinks, have an  application/rdf+xml serialization as well. Would simplify export to MODS for preservation and to TEI header